electrumsv/networks.py

```diff
@@ -248,10 +245,9 @@ class SVRegTestnet(object):
     ADDRTYPE_P2SH = 196
     CASHADDR_PREFIX = "bchtest"
     DEFAULT_PORTS = {'t': '51001', 's': '51002'}
-    DEFAULT_SERVERS_ELECTRUMX = read_json_dict('servers_regtest.json')
     DEFAULT_SERVERS_MAPI = read_json_dict('mapi_servers_regtest.json')
     DEFAULT_SERVERS_API = read_json_dict('api_servers_regtest.json')
-    GENESIS = "000000000933ea01ad0ee984209779baaec3ced90fa3f408719526f8d77f4943"
+    GENESIS = "0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206"
```

Comment:
I don't know why this is changing.

Reply:
Because the genesis block hash is `0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206` on RegTest. You can see for yourself with `electrumsv-sdk node getblockbyheight 0`